packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7669,6 +7709,7 @@ describe('plugin-meetings', () => {
               meetingId: meeting.id,
             }
           );
+          cleanUpSpy.restore();
```

Comment:
What does this help us with?

Reply:
In tests for '#destroy', we are stubbing the cleanUp method, if we dont restore it here then we get error while stubbing in the destroy test.  'Trying to wrap a method that is already a stub'